pkg/kev/converter/kubernetes/project_service.go

```diff
@@ -34,6 +34,17 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+// enabled returns Bool telling Kev whether app component is enabled/disabled
+func (p *ProjectService) enabled() bool {
+	if val, ok := p.Labels[config.LabelComponentEnabled]; ok {
+		if val == "false" {
+			return false
+		}
```

Comment:
This can introduce some subtle issues, e.g. misspelling false will trigger that the service is `enabled`.

IMO, we should parse the value to ensure we have a bool and act accordingly. Maybe something like `strconv.ParseBool()` can help?